src/internal/internal-types.ts

```diff
@@ -140,15 +142,39 @@ class Event {
  * @internal
  */
 export class EthersEvent extends Event {
-  get address(): string | null {
+  get fromAddress(): string | string[] | undefined {
```

Comment:
Add a comment explaining what's happening in this class as well please!

Reply:
Specifically what fields and format (toAddress, fromAddress, hashes) are being supported by this code